### PR TITLE
Embedded YT Player

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -160,19 +160,10 @@ body{
...
-}
-
-td{
-  border-style: solid;
-  border-width: 3px;
-  border-color: black;
-  color: #e67e22;
-  font-family: 'Poppins', sans-serif;
+#player{
+  margin-top: 30px;
```
Comment (TimAshton) [inline assets/css/style.css]: can use margin: 30px; instead of all 4 sides.

File: assets/javascript/app.js [hunk 2]
```diff
@@ -149,6 +149,14 @@ var movieArray = [
   { name: 'Wives Under Suspicion', year: 1938 }
 ];
 
+movieArray.sort(function (a,b){
```
Comment (TimAshton) [inline assets/javascript/app.js]: What does this do?

Comment (ghost) [reply assets/javascript/app.js]: Nothing for now, I forgot to comment it out.

**Comment by WesleyClements** [comment]
master is more up to date than this branch

**Pull request closed without merging**